lib/hotel_manager.rb

```diff
@@ -0,0 +1,123 @@
+require 'date'
+require_relative 'room'
+require_relative 'reservations'
+require_relative 'hotel_block'
+
+
+module Hotel
+  class HotelManager
+    attr_accessor :rooms, :blocks
+
+    def initialize(rooms:,blocks:)
+      @rooms = @Hotel::Room.list_of_rooms
```

Comment:
When you are inside the scope of `module Hotel` you don't need to do `Hotel::`. The `@Hotel` ends up causing a bunch of warnings.